README.md

```diff
@@ -2,7 +2,7 @@
 <img src="https://github.com/google-research/ott/raw/master/docs/logoOTT.png" alt="logo"  width="150"></img>
 </div>
 
-# Optimal Transport Tools (OTT), A toolbox for everything Wasserstein.
+# Optimal Transport Tools (OTT), A toolbox for everything Kantorovich-Rubenstein
```

Comment:
Thanks!!

so Kantorovich Rubinstein is usually a norm! and it's very associated to what other people would call Wasserstein 1. It's a norm on algebraic measures with 0 mass, or, alternatively, the W_1 distance between their negative / positive parts.

I agree the debate is a bit open in terms of naming. The best alternative is probably "Monge-Kantorovich" but that's tool long.

I think I'll side with Villani (talks about it in his book) and just side with common usage! But we will make sure to increase the use of Monge and Kantorovich in the doc and elsewhere :) 

thanks for the other typos about not capitalizing Sinkhorn.

Reply:
Probably a minor issue, but one of the postdocs at McGill [apparently](https://www.youtube.com/watch?v=UgRIyy21Ucw&t=4013s) contacted [Prof. Vaseršteĭn](http://www.personal.psu.edu/lxv1/), who acknowledges Kantorovich et al. Although Monge-Kantorovich is less pronounceable, Wasserstein is definitely a misnomer. It might be worth confirming with the relevant parties.

Anyhow, thanks for publishing this project, the documentation looks fantastic! Looking forward to trying it out. :)

Reply:
Thanks Breandan. The fact we ended up stressing the name Wasserstein in OT theory is clearly the result of a curious turn of events. This happened a couple of decades ago though, and Wasserstein is now commonly used to describe a few aspects of OT. I do think it is more fair to give more credit to Monge / Kantorovich, and we can certainly keep that in mind for future changes.